instal libgettextlib.a before removing it

In a multiple job build, Makefile can simultaneously
be installing and removing libgettextlib.a.  We serialize
the operations.

Upstream-Status: Pending

Signed-off-by: Joe Slater <jslater@windriver.com>

--- a/gettext-tools/gnulib-lib/Makefile.am
+++ b/gettext-tools/gnulib-lib/Makefile.am
@@ -57,6 +57,10 @@ endif
 # Rules generated and collected by gnulib-tool.
 include Makefile.gnulib
 
+# defined in Makefile.gnulib but missing this dependency
+#
+install-exec-clean: install-libLTLIBRARIES
+
 # Which classes to export from the shared library.
 MOOPPFLAGS += --dllexport=styled_ostream
 
